apps/src/templates/teacherDashboard/AssignmentSelector.jsx

```diff
@@ -1,328 +0,0 @@
-import _ from 'lodash';
-import PropTypes from 'prop-types';
-import React, {Component} from 'react';
-
-import fontConstants from '@cdo/apps/fontConstants';
-import {
-  CourseOfferingCategories,
-  ParticipantAudiencesByType,
-} from '@cdo/apps/generated/curriculum/sharedCourseConstants';
-import i18n from '@cdo/locale';
-
-import {translatedCourseOfferingCategories} from './AssignmentSelectorHelpers';
```

Comment:
Should we rename this file or move it into `CourseOfferingEditor` (the only other place it is used)?